### PR TITLE
Hero Tile layout 

### PR DESCRIPTION
**Pull request by nileshgulia1**
Addresses #289  Need to scale images. I'm guessing css `object-fit` property here, though it needs some minor fixes also. I'll be on it.

### CODE REVIEW

**Comment by nileshgulia1** [comment]
Fixes Done! Please review..

File: src/components/manage/Tiles/Text/Edit.jsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ import trashSVG from '../../../../icons/delete.svg';
 import addSVG from '../../../../icons/circle-plus.svg';
 import cameraSVG from '../../../../icons/camera.svg';
 import videoSVG from '../../../../icons/videocamera.svg';
+import cdSVG from '../../../../icons/cd.svg';
```
Comment (sneridagh) [inline src/components/manage/Tiles/Text/Edit.jsx]: I would use table-header.svg better. Requesting a better icon to Albert.

File: src/config/Tiles.jsx [hunk 2]
```diff
@@ -29,6 +30,10 @@ const messagesTiles = defineMessages({
     id: 'video',
     defaultMessage: 'Video',
   },
+  hero: {
```
Comment (sneridagh) [inline src/config/Tiles.jsx]: Same as the Google Maps one, please put it on the second level tiles.

**Comment by sneridagh** [comment]
Please also, rebase and put the changelog in the right place.

**Comment by sneridagh** [comment]
@nileshgulia1 Please take care of this too.

**Comment by sneridagh** [comment]
@nileshgulia1 please can you merge from master and resolve the conflicts? I will take a look then!

**Comment by nileshgulia1** [comment]
@sneridagh I just merge with master.

**Comment by sneridagh** [comment]
This is the edit mode:
screenshot 2019-03-04 at 15 24 37


This is the view:

screenshot 2019-03-04 at 15 24 45

They should be the same, so you have almost the same "What You See Is What You Get" feel, can you take a look and try them both looks like the same? Including the empty one.

screenshot 2019-03-04 at 15 28 50

 Also, did we decide to have Title/UnderTitle/Description? 

The placeholders are still in German ;)

/cc @tisto

**Comment by tisto** [comment]
@nileshgulia1 what's the status here? Did you fix the issues raised by Victor?

**Comment by nileshgulia1** [comment]
@tisto. Yes I changed the maxWidth to 50% now we can see the image with
title nd subtitle. Can you please review it now?

**Comment by tisto** [comment]
4) Hero tile title and subtitle do not have any left margin/padding. Subtitle is same font weight and font size as title -> reduce size and font weight of subtitle.

add document 3

**Comment by tisto** [comment]
@sneridagh do we have an icon from @albertcasado for a hero tile with the image on the left side? Maybe we should add both hero tiles to Volto for more options...

**Comment by nileshgulia1** [comment]
@tisto I cannot able to reproduce the image alignment, on my side its like:
Screenshot_2019-03-11 Add pagevisual

**Comment by tisto** [comment]
@nileshgulia1 please see my comments:

Add Document (2)

The image issue hasn't been fixed either. Now the image is 100% height but does not keep its ratio:

Add Document (3)

**Comment by tisto** [comment]
Screen for the "Hero Tile with Top image":

tile hero imagetop

(Note that this would be a second tile, not the one we are currently working on)

**Comment by tisto** [comment]
@nileshgulia1 looks a lot better now. I would like to amend one detail though:

Add Document (4)

We should also use a light grey background as we have here:

tile hero imageleft

File: src/components/manage/Tiles/Hero/Edit.jsx [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+                      url: '',
+                    })
+                  }
+                >
+                  <Icon name={clearSVG} size="24px" color="#e40166" />
+                </Button>
+              </Button.Group>
+            </div>
+          )}
+        <div className="product-hero">
```
Comment (sneridagh) [inline src/components/manage/Tiles/Hero/Edit.jsx]: "product-hero" comes from Zeelandia project, please use a more descriptive class. We can think in a broader solution, like `tile-inner-wrapper`?

Comment (nileshgulia1) [reply src/components/manage/Tiles/Hero/Edit.jsx]: Hm. I created and manually amend changes on it . I will change the same.

**Comment by sneridagh** [comment]
I can't barelly see the placeholder text now:

Screenshot 2019-03-13 at 18 49 24

and the color is very dark, please change it to pastanaga @tealBackground variable.

The padding in the text body wrapper is not sufficient for me, increase it to match the mockup that Timo pasted above.

**Comment by nileshgulia1** [comment]
@sneridagh Done!

**Comment by tisto** [comment]
@nileshgulia1 looks good to me. One small amendment:

Add Document (5)

@sneridagh we still have the problem with the icon. I think we need the correct icon before we can merge and release. :(

File: theme/themes/pastanaga/globals/site.overrides [hunk 4]
```diff
@@ -677,9 +683,9 @@ body {
 
   .toolbar {
     position: absolute;
-    z-index: 1;
+    z-index: 2!important;
```
Comment (sneridagh) [inline theme/themes/pastanaga/globals/site.overrides]: The important is not required.

File: theme/themes/pastanaga/globals/site.overrides [hunk 5]
```diff
@@ -29,6 +29,12 @@ body {
   background-color: #a6f3a6;
 }
 
+.product-hero {
+  z-index: '123';
+  position: 'relative';
+  display: 'flex';
+}
+
```
[the remark this answers is not in the document]
Comment (nileshgulia1) [reply theme/themes/pastanaga/globals/site.overrides]: crap. sorry, I was testing with those, my bad.

File: theme/themes/pastanaga/globals/site.overrides (hunk 4 above)
Comment (nileshgulia1) [reply theme/themes/pastanaga/globals/site.overrides]: Without, its not working....

File: theme/themes/pastanaga/globals/site.overrides [hunk 6]
```diff
@@ -871,12 +877,53 @@ body {
   margin-top: 20px;
 }
 
+.inline-tile-wrapper {
```
[the remark this answers is not in the document]
Comment (nileshgulia1) [reply theme/themes/pastanaga/globals/site.overrides]: ok

File: theme/themes/pastanaga/globals/site.overrides [hunk 7]
```diff
@@ -679,7 +685,7 @@ body {
     position: absolute;
     z-index: 1;
     top: -32px;
-    left: 50%;
+    left: 25%;
```
[the remark this answers is not in the document]
Comment (nileshgulia1) [reply theme/themes/pastanaga/globals/site.overrides]: This is for the `clearSVG` icon,right?. We have to centre it according to timo's review.

File: theme/themes/pastanaga/globals/site.overrides (hunk 4 above)
Comment (sneridagh) [reply theme/themes/pastanaga/globals/site.overrides]: it does:
Screenshot 2019-03-14 at 14 26 08

**Comment by tisto** [comment]
@nileshgulia1 @sneridagh image tile is broken, no way to add a new paragraph in the description text of the tile, if you add a very long word without any blanks the word will bleed out of the tile.

**Comment by sneridagh** [comment]
@nileshgulia1 I finished the tile. Nilesh, please take a look at the changes of my last commits. I had to fix some changes that were still due to implement from the last reviews. I fixed and corrected some other things, and removed leftovers from tests, and dangling imports and ESlint errors.

@tisto I also added the support for cursor navigation. The ability to add a paragraph using shift+enter will have to wait to implement it on the normal text tiles... still don't know how to do it. The break-word if we have long words is also fixed. Take a look and tell me if we can merge.

We still need the final icon.

**Comment by nileshgulia1** [comment]
@sneridagh Thanks for amending on top of it. Yeah I have understood the amendments, specially those `CSS` changes and the `tile focus` logic.

**Pull request merged**